cosmos/sge-network.json

```diff
@@ -0,0 +1,49 @@
+{   "chainId": "sge-network-4",
+    "chainName": "SGE Network Testnet",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/sgenet/chain.png",

```

Comment:
Hi! Please write the right directory name. (sgenet -> sge-network)

Reply:
@editaahn Done.